# 一帧图像里每辆车检测的装甲板信息

uint8 armorClass                      # 装甲板的类别：10-14--红1~红5，1-5--蓝1~蓝5
uint32 armorX                         # 装甲板的中心点x轴图像坐标
uint32 armorY                         # 装甲板的中心点y轴图像坐标
uint32 armorWidth                     # 装甲板的图像宽度
uint32 armorHeight                    # 装甲板的图像高度